src/app/features/public-form/edit-prefilled-values/edit-prefilled-values.component.ts

```diff
@@ -0,0 +1,126 @@
+import { CommonModule } from "@angular/common";
+import { Component, OnInit, inject } from "@angular/core";
+import {
+  FormArray,
+  FormBuilder,
+  ReactiveFormsModule,
+  Validators,
+} from "@angular/forms";
+import { DefaultValueOptionsComponent } from "app/core/admin/admin-entity-details/admin-entity-field/default-value-options/default-value-options.component";
+import { EntityRegistry } from "app/core/entity/database-entity.decorator";
+import { EditComponent } from "app/core/entity/default-datatype/edit-component";
+import { EntityConstructor } from "app/core/entity/model/entity";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatSelectModule } from "@angular/material/select";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { FormConfig } from "app/core/entity-details/form/form.component";
+import { DefaultValueConfig } from "app/core/entity/schema/default-value-config";
+import { HelpButtonComponent } from "app/core/common-components/help-button/help-button.component";
+import { MatButtonModule } from "@angular/material/button";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { FormFieldConfig } from "app/core/common-components/entity-form/FormConfig";
+import { EntityFieldSelectComponent } from "app/core/entity/entity-field-select/entity-field-select.component";
+
+@Component({
+  selector: "app-edit-prefilled-values",
+  standalone: true,
+  imports: [
+    DefaultValueOptionsComponent,
+    CommonModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatSelectModule,
+    MatTooltipModule,
+    HelpButtonComponent,
+    FontAwesomeModule,
+    MatButtonModule,
+    EntityFieldSelectComponent,
+  ],
+  templateUrl: "./edit-prefilled-values.component.html",
+  styleUrls: ["./edit-prefilled-values.component.scss"],
+})
+export class EditPrefilledValuesComponent
+  extends EditComponent<FormFieldConfig[]>
+  implements OnInit
+{
+  entityConstructor: EntityConstructor;
+  formConfig: FormConfig;
+  FormFieldConfig: FormFieldConfig;
+  defaultValue: DefaultValueConfig;
+
+  private entities = inject(EntityRegistry);
+
+  private fb = inject(FormBuilder);
```

Comment:
Currently, the tab/form can be saved even when some of the fields within this component show as invalid (e.g. required field empty). This should integrate with the EditComponent's formControl and set it invalid when necessary